generators/app/bitriseUtils.js

```diff
@@ -0,0 +1,47 @@
+const fs = require('fs');
+
+function isNotEmpty(key, value) {
+  const message = `The field ${key} in bitriseInfo.json file is required to run the script`;
+  if (
+    ((!value && typeof value === 'string') || value === null || value === undefined) &&
+    key !== 'projectPath' &&
+    key !== 'projectName'
+  ) {
+    return message;
+  }
+  return '';
+}
```

Comment:
- Could value be any type different of string? If not you could just ask for `!value` and `key !==...`.
- You could make 
```
return condition ? `The field ${key} in bitriseInfo.json file is required to run the script` : '';
```

Reply:
Yes, value could be boolean, so in that case I have to ask